tools/python-integration-tests/deployment.py

```diff
@@ -50,8 +50,16 @@ def get_posixAccount_info(self):
                 self.project_id = account['accountId']
                 self.username = account['username']
 
+    def set_deployment_name(self):
+        BUILD_ID = os.environ.get('BUILD_ID_FULL')
+        if BUILD_ID:
+            self.deployment_name = BUILD_ID[:6]
+        else:
+            self.deployment_name = str(uuid.uuid4())[:6]
+
     def set_deployment_variables(self):
         self.workspace = os.path.abspath(os.getcwd().strip())
+        self.deployment = self.set_deployment_name()
```

Comment:
This is very strange chain of execution., why do you override `deployment` with `None`? Either:

Recommended:
```py
self.deployment_name = generate_uniq_deployment_name()
```

OR (unrecommended)

```py
self.set_deployment_name()
```

Reply:
Ah yes this is very strange. I had multiple path in my head when I wrote this. Cleaned up according to recommended.